src/index.ts

```diff
@@ -42,13 +45,18 @@ type ResponseFormat = 'json' | 'text' | 'html';
    app.use(errorReporter());
    ```
  */
-export function errorReporter() {
+export function errorReporter(options?: ErrorReporterOptions) {
```

Comment:
Create a local `const realOptions: ErrorReporterOptions = {...defaultOptions, ...options}` (of course, you need to define the default options somewhere).

This allows you to use non-optional types everywhere else in the internal code.

Reply:
note: you might have to use `Object.assign` if TypeScript complains about the spread operator.

Reply:
fixed